Example_Systems/PiecewiseFuel_CO2_Example/README.md

```diff
@@ -0,0 +1,17 @@
+# PiecewiseFuel_CO2
+
+**PiecewiseFuel_CO2** is a 24 hr example and contains only one zone. It is condensed for easy and quick testing of CO2, biomass, and piecewise fuel usage related functions of the GenX. This testing 
+system only includes natural gas ccs, wind, and biomass with ccs, all set at a fixed initial 
+capacity, and does not allow for building additional capacity."
+
+To run the model, first navigate to the example directory at `GenX/Example_Systems/PiecewiseFuel_CO2`:
+
+`cd("Example_Systems/PiecewiseFuel_CO2")`
+   
+Next, ensure that your settings in `GenX_settings.yml` are correct. The default settings use the solver HiGHS (`Solver: HiGHS`). Other optional policies include minimum capacity requirements, a capacity reserve margin, and more. A mass-based carbon cap of 0 t CO<sub>2</sub> (net-zero) is specified in the `CO2_cap.csv` input file.
```

Comment:
> Other optional policies include minimum capacity requirements, a capacity reserve margin, and more.

These policies could certainly be added, but the relevant files are not part of this case. I think this sentence can be deleted.